src/instance.ts

```diff
@@ -415,17 +422,6 @@ Please use the format 'my-instance' or '${bigtable.projectName}/instances/my-ins
       reqOpts.cluster!.encryptionConfig = options.encryption;
     }
 
-    if (options.location) {
-      reqOpts.cluster!.location = Cluster.getLocation_(
-        this.bigtable.projectId,
-        options.location
-      );
-    }
-
-    if (options.nodes) {
```

Comment:
why is this deleted?

Reply:
`cluster.serveNodes` is now provided by the call to `ClusterUtils.getClusterBaseConfig` above. This change is a refactor so that this function doesn't get too verbose.